content/components/tab-nav.mdx

```diff
@@ -18,26 +19,20 @@ export default ComponentLayout
 
 ## Usage
 
-A tab nav is used to organize related content into different views within the same context.
+A tab nav lets users navigate between related views in the same context. Each "tab" links to a different view that is shown when a user selects a tab.
+
+A tab nav is always a set of links, and each tab should have a unique URL.
```

Comment:
What do you think of having a sentence here that includes a link to tab panels, and the tab panels usage section having a sentence that includes a link to tab nav?

My thinking is it could help pre-empt folks who don't know the difference between the two components' use cases and provides a link for them to self-serve.